### PR TITLE
add go-upower-notify

### PR DESCRIPTION
**Pull request by obadz**
Adding go-upower-notify
Upgrading go-dbus was a prereq.
Also adding yeshup which is very useful to launch upower-notify.
###### Things done:
- [x] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- [x] Built on platform(s): NixOS / ~~OSX~~ / Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
###### More

---

_Please note, that points are not mandatory, but rather desired._

### CODE REVIEW

File: pkgs/tools/system/yeshup/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  rev = "5461a8f957c686ccd0240be3f0fd8124d7381b08";
+
+  src = fetchFromGitHub {
+    owner = "RhysU";
+    repo  = "yeshup";
+    inherit rev;
+    sha256 = "1wwbc158y46jsmdi1lp0m3dlbr9kvzvwxfvzj6646cpy9d6h21v9";
+  };
+
+  phases = [ "unpackPhase" "buildPhase" "installPhase" ];
```
Comment (lethalman) [inline pkgs/tools/system/yeshup/default.nix]: You don't need this, do you?

Comment (obadz) [inline pkgs/tools/system/yeshup/default.nix]: You are correct

**Comment by lethalman** [comment]
Travis passed, thanks.

**Comment by obadz** [comment]
:+1:

**Pull request merged**